src/Aspire.Dashboard/Components/ResourcesGridColumns/StateColumnDisplay.razor

```diff
@@ -89,7 +96,14 @@ else
 }
 else
 {
-    <span>@Resource.State.Humanize()</span>
+    if (Resource.HealthState is null)
```

Comment:
Do we want to display `Unknown` in the UI, or should we filter that out here along with `null`?

Reply:
Made a change, it will never show unknown.